inst/rmarkdown/templates/ims/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,422 @@
+\item This is the second item of an enumerated list.
+\end{longlist}
+
+The following is an example of an *enumerated* list, two levels deep.
+\begin{longlist}
+\item[1.]
+This is the first item of an enumerated list.  Each item
+in the list is marked with a ``tick.''.  The document
+style determines what kind of tick mark is used.
+\item[2.]
+This is the second item of the list.  It contains another
+list nested inside of it.
+\begin{longlist}
+\item
+This is the first item of an enumerated list that
+is nested within.  
+\item
+This is the second item of the inner list.  \LaTeX\
+allows you to nest lists deeper than you really should.
+\end{longlist}
+This is the rest of the second item of the outer list.
+\item[3.]
+This is the third item of the list.
+\end{longlist}
+
+## Punctuation
+
+Dashes come in three sizes: a hyphen, an intra-word dash like "$U$-statistics" or "the time-homogeneous model";
+a medium dash (also called an "en-dash") for number ranges or between two equal entities like "1--2" or "Cauchy--Schwarz inequality";
+and a punctuation dash (also called an "em-dash") in place of a comma, semicolon,
+colon or parentheses---like this.
+
+Generating an ellipsis \ldots\ with the right spacing
+around the periods requires a special command.
+
+## Citation
+
+Simple author and year cite: @billingsley2013convergence.
+Multiple bibliography items cite: \cite{billingsley2013convergence,bourbaki1966general} or [@billingsley2013convergence; @bourbaki1966general].
+Author only cite: \citeauthor{ethier1985markov}.
+Year only cite: \citeyear{prokhorov1956convergence} or [-@prokhorov1956convergence].
+
+# Fonts
+
+Please use text fonts in text mode, e.g.:
+\begin{itemize}
+\item[]\textrm{Roman}
+\item[]\textit{Italic}
+\item[]\textbf{Bold}
+\item[]\textsc{Small Caps}
+\item[]\textsf{Sans serif}
+\item[]\texttt{Typewriter}
+\end{itemize}
+Please use mathematical fonts in mathematical mode, e.g.:
+\begin{itemize}
+\item[] $\mathrm{ABCabc123}$
+\item[] $\mathit{ABCabc123}$
+\item[] $\mathbf{ABCabc123}$
+\item[] $\boldsymbol{ABCabc123\alpha\beta\gamma}$
+\item[] $\mathcal{ABC}$
+\item[] $\mathbb{ABC}$
+\item[] $\mathsf{ABCabc123}$
+\item[] $\mathtt{ABCabc123}$
+\item[] $\mathfrak{ABCabc123}$
+\end{itemize}
+Note that \verb|\mathcal, \mathbb| belongs to capital letters-only font typefaces.
+
+# Notes
+
+Footnotes[^1]
+pose no problem.[^2]
+
+[^1]: This is an example of a footnote.
+[^2]: Note that footnote number is after punctuation.
+
+# Quotations
+
+Text is displayed by indenting it from the left margin. There are short quotations
+
+> This is a short quotation.  It consists of a
+> single paragraph of text.  There is no paragraph
+> indentation.
+
+and longer ones.
+
+<!-- The following syntax works with Rmarkdown >= 2.7 -->
+<!-- ::: {.environment latex=true} -->
+<!--   code -->
+<!-- ::: -->
+<!-- It's equivalent to  -->
+<!-- \begin{environment} -->
+<!--   % code -->
+<!-- \end{environment} -->
```

Comment:
You are right. I went a bit ahead of what is available. 
The current syntax that works is : 
````markdown
::: {.environment data-latex=""}
   code
:::
````
Doc: https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html

We added a simplified version in dev **rmarkdown**. We may use the current working version instead and change when 2.7 in on CRAN. What do you think ? 

Thanks for adding this note in the skeleton!

Reply:
Thanks for the hint. Now it uses the current working version

Reply:
It's updated to work with the current syntax. Thanks for pointing out the documentation.